### PR TITLE
Handle withFortescue parameter

### PR DESCRIPTION
**Pull request by Meklo**

### CODE REVIEW

File: src/main/java/org/gridsuite/shortcircuit/server/utils/ShortcircuitUtils.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package org.gridsuite.shortcircuit.server.utils;
```
Comment (FranckLecuyer) [inline src/main/java/org/gridsuite/shortcircuit/server/utils/ShortcircuitUtils.java:1]: Add license and header

File: src/main/java/org/gridsuite/shortcircuit/server/entities/FaultResultEntity.java [hunk 2]
```diff
@@ -62,7 +62,37 @@ public class FaultResultEntity {
...
+    @AttributeOverride(name = "positiveAngle", column = @Column(name = "fortescue_voltage_positive_angle"))
+    @AttributeOverride(name = "zeroAngle", column = @Column(name = "fortescue_voltage_zero_angle"))
+    @AttributeOverride(name = "negativeAngle", column = @Column(name = "fortescue_voltage_negative_angle"))
+    @AttributeOverride(name = "magnitudeA", column = @Column(name = "fortescue_voltage_magnitude_a"))
+    @AttributeOverride(name = "magnitudeB", column = @Column(name = "fortescue_voltage_magnitude_b"))
+    @AttributeOverride(name = "magnitudeC", column = @Column(name = "fortescue_voltage_magnitude_c"))
+    @AttributeOverride(name = "angleA", column = @Column(name = "fortescue_voltage_angle_a"))
+    @AttributeOverride(name = "angleB", column = @Column(name = "fortescue_voltage_angle_b"))
+    @AttributeOverride(name = "angleC", column = @Column(name = "fortescue_voltage_angle_c"))
+    private FortescueResultEmbeddable fortescueVoltage;
```
Comment (FranckLecuyer) [inline src/main/java/org/gridsuite/shortcircuit/server/entities/FaultResultEntity.java:93]: where is this field 'fortescueVoltage' used ?

File: src/main/java/org/gridsuite/shortcircuit/server/entities/FaultResultEntity.java [hunk 3]
```diff
@@ -88,4 +120,8 @@ public boolean equals(Object o) {
     public int hashCode() {
         return getClass().hashCode();
     }
+
+    public double getPositiveMagnitude() {
+        return this.getFortescueCurrent() != null ? this.getFortescueCurrent().getPositiveMagnitude() : Double.NaN;
```
Comment (FranckLecuyer) [inline src/main/java/org/gridsuite/shortcircuit/server/entities/FaultResultEntity.java:125]: Is it normal to only access 'fortescueCurrent' and never 'fortescueVoltage' ?

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]

File: src/main/java/org/gridsuite/shortcircuit/server/entities/FaultResultEntity.java (hunk 2 above)
Comment (Meklo) [reply src/main/java/org/gridsuite/shortcircuit/server/entities/FaultResultEntity.java:93]: It is not currently used but is it asked in the US to store all the retrieved data for future usage, I confirmed this with a PO prior to taking this approach

File: src/main/java/org/gridsuite/shortcircuit/server/entities/FaultResultEntity.java (hunk 3 above)
Comment (Meklo) [reply src/main/java/org/gridsuite/shortcircuit/server/entities/FaultResultEntity.java:125]: Same as above, It was asked to store all retrieved data but it not everything is used for the time being

File: src/main/java/org/gridsuite/shortcircuit/server/utils/ShortcircuitUtils.java (hunk 1 above)
Comment (Meklo) [reply src/main/java/org/gridsuite/shortcircuit/server/utils/ShortcircuitUtils.java:7]: Done

**Review by YenguiSeddik — APPROVED** [review]

**Review by FranckLecuyer — APPROVED** [review]

**Pull request merged**